website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -814,6 +819,20 @@ In addition to ACLs, in Consul 0.9.0 and later, the agent must be configured wit
 [`enable_local_script_checks`](/docs/agent/config/config-files#enable_local_script_checks)
 set to `true` in order to enable script checks.
 
+### Reading Imported Services
+
+Service rules affect read access to services exported by [`exported-services` configuration entries](/docs/connect/config-entries/exported-services#reading-services), including services exported between [cluster peerings](/docs/connect/cluster-peering) or [admin partitions](/docs/enterprise/admin-partitions).
+Read access to all imported services is granted when either of the following rule sets are attached to a token:
+- `service:write` is granted to any service. 
+- `service:read` and `node:read` is granted to all services and nodes, respectively.
```

Comment:
So this actually depends on the endpoint. Currently the only public endpoints that can be used to query data imported from peers `/health/<service>|<connect>` return both node and service data. This means you need to be able to read all nodes and services to read imports. 

However, if we add support to query by peer for other catalog endpoints in the future then this is going to end up being out of date. 

To guard against that I think this block should only talk about `service:read` and the `Node Rules` section above should be a mirror of this one, but only referencing `node:read`. As opposed to how it currently links to this section.

What do you think?

Reply:
That makes sense, although I might proactively link between the sections and note that the permissions are related. You would know better than I do, but AFAIK, in practice, many endpoints return CheckNodeServices, and these permissions are implicitly tied together.